### PR TITLE
53 backend populate with more tags users and projects

### PR DESCRIPTION
**Pull request by timabloom**

### CODE REVIEW

File: Backend/Database/DbSeeder.cs [hunk 1]
```diff
@@ -24,18 +21,34 @@ public static (List<Project>, List<User>, List<Tag>) GenerateProjects(int count)
...
-                .RuleFor(t => t.TagValue, v => tags[index1])
-                .RuleFor(t => t.IsSkill, v => true);
+                fakedTags.Add(tagFaker.Generate());
+                skills.RemoveAt(index);
+            }
 
-            var tagFake2r = new Faker<Tag>()
-                .RuleFor(t => t.TagValue, v => tags[index2])
-                .RuleFor(t => t.IsSkill, v => false);
+            List<string> interests = ["Data", "Art", "C#", "Tim", "JS", "Java"];
```
Comment (rorychatt) [inline Backend/Database/DbSeeder.cs:41]: how about we make these seeded lists contain different values from skills list?

File: Backend/Database/DatabaseContext.cs [hunk 2]
```diff
@@ -77,7 +77,7 @@ internal List<ProjectResponse> GetProjectsByFilter(string[]? skills, string[]? i
         return (DbErrorStatusCodes.Ok, projects);
     }
 
-    internal bool PopulateProjects(int count = 10)
+    internal bool PopulateProjects(int count = 1000)
```
Comment (rorychatt) [inline Backend/Database/DatabaseContext.cs:80]: 1000 might be overkill, maybe 100 would be better or sth

**Review by rorychatt — COMMENTED** [review]
LGTM

File: Backend/Database/DatabaseContext.cs (hunk 2 above)
Comment (timabloom) [reply Backend/Database/DatabaseContext.cs:80]: Oscar wished for 1000 and yes 100 is enough also

File: Backend/Database/DbSeeder.cs (hunk 1 above)
Comment (timabloom) [reply Backend/Database/DbSeeder.cs:41]: I don't see the need or effect it will have on our features

**Pull request merged**